external/dynamo/.env.example

```
@@ -84,6 +84,18 @@ export DYNAMO_HTTP_PORT=8000
 
+# =============================================================================
+# OPTIONAL VARIABLES - HiCache Configuration
+# =============================================================================
+
+# Enable hierarchical cache on SGLang workers (required for cache_control pin_prefix)
+export DYNAMO_ENABLE_HIERARCHICAL_CACHE=true
+# HiCache configuration (only applies when DYNAMO_ENABLE_HIERARCHICAL_CACHE=true)
+# CPU pool size as a multiple of GPU KV cache pool (default: 1.0 = same size)
+export DYNAMO_HICACHE_RATIO=1.0
+# Write policy: write_through (copy to CPU on insert) or write_back (copy on evict)
+export DYNAMO_HICACHE_POLICY=write_through
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The DYNAMO_HICACHE_POLICY key should go before the DYNAMO_HICACHE_RATIO key

(UnorderedKey)